lib/puppet/provider/ini_setting/splunk.rb

```diff
@@ -13,7 +13,7 @@ def self.file_path
     File.join(@file_path, file_name)
   end
 
-  def self.set_file_path(path)
+  def self.set_file_path(path) # rubocop:disable Style/AccessorMethodName
```

Comment:
According to the rubocop docs we should rename this to 

```ruby
def self.file_path=(path)
```

And then change the references to this method in splunk_config

https://github.com/voxpupuli/puppet-splunk/blob/master/lib/puppet/type/splunk_config.rb#L95-L104

eg;

```
Puppet::Type.type(res_type).provider(:ini_setting).file_path=(self[:server_confdir])
```